Remotion/Web/Development.WebTesting.IntegrationTests.Infrastructure/GenericPageTestCaseFactoryBase`.cs

```diff
@@ -57,8 +60,13 @@ protected void PrepareTest ([NotNull] GenericPageTestMethodAttribute attribute,
       ArgumentUtility.CheckNotNull ("helper", helper);
       ArgumentUtility.CheckNotNullOrEmpty ("control", control);
 
+      if (_webApplicationRoot == null)
+      {
+        _webApplicationRoot = new Lazy<Uri> (() => HostnameResolveHelper.ResolveHostname (new Uri (helper.TestInfrastructureConfiguration.WebApplicationRoot)));

```

Comment:
Should we leave a comment explaining why we're resolving it manually here as well?